### DISCUSSION TITLE
New module for joining LineStrings together

### DISCUSSION THREAD

**Discussion by andrewharvey** [Turfjs/turf#2787] [Feature proposals]
I couldn't find anything existing to join LineStrings together so I put something together https://github.com/andrewharvey/join-lines. Like dissolve/union but for LineStrings.

If I get time I might try to work this into a turf module.

**Comment by smallsaucepan** [comment]
Hi @andrewharvey. Note you're still tweaking your project. Do you still have an interest in adding it to Turf?

**Comment by andrewharvey** [comment]
I would like to see it in turf, but I don't have time to contribute it directly as a module right now. While it seems to work okay, it hasn't been tested more rigorously, so it also depends from the Turf point of view how stable you'd want your modules to be...
